Source/LibationWinForms/grid/SortableBindingList1.cs

```diff
@@ -0,0 +1,87 @@
+﻿using Dinah.Core.DataBinding;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+
+namespace LibationWinForms
+{
+	internal class SortableBindingList1<T> : BindingList<T> where T : class, IMemberComparable, IHierarchical<T>
```

Comment:
Should I just remove the sortable binding stuff from dinah.core?

Reply:
Probably. If it's going to be useful to you for other projects then keep it, but at this point it's so tailored to the libation implementation that it doesn't really make sense for it to be generic anymore.